chutney/engine/src/main/java/com/chutneytesting/ExecutionConfiguration.java

```diff
@@ -65,7 +71,7 @@ public class ExecutionConfiguration {
 
     private final SpelFunctions spelFunctions;
     private final Set<StepExecutionStrategy> stepExecutionStrategies;
-
+    private static ObjectMapper objectMapper;
```

Comment:
Two points :
* Rename it as reportObjectMapper to underline the use of it.
* Instanciation should be done as the others, i.e. a create method called in constructor.

Reply:
or as its static directly in attribute declaration